sgkit/stats/hwe.py

```diff
@@ -137,6 +139,12 @@ def hardy_weinberg_test(
         where `N` is equal to the number of variants and the 3 columns contain
         heterozygous, homozygous reference, and homozygous alternate counts
         (in that order) across all samples for a variant.
+    merge : bool, optional
+        If True (the default), merge the input dataset and the computed
```

Comment:
I don't like the idea of duplicating this block of text in every function. Maybe we could refer to a section of the docs discussing what merge is, something like "If True (the default) merge the input dataset and the computed output variables into a single dataset. See :ref:`sec_dataset_merge` for more details".

Or, we could try to define a sphinx substitution or something?

Reply:
I prefer having a reference. Implemented in https://github.com/pystatgen/sgkit/pull/250/commits/cfc5e2f00d16d282e581ce6b707f08d43e161c63.